src/upstream/rss3/mod.rs

```diff
@@ -16,6 +16,7 @@ use async_trait::async_trait;
 use chrono::{DateTime, NaiveDateTime};
 use futures::future::join_all;
 use http::uri::InvalidUri;
+use hyper::{Body, Client, Method, Request};
```

Comment:
these imports for ?

Reply:
Some automatic imports, I'll delete them.